gpMgmt/bin/gpstart

```diff
@@ -55,7 +55,8 @@ class GpStart:
                  skip_heap_checksum_validation=False,
                  fts_hosts=None,
                  etcd_hosts=None,
-                 is_external_fts=False
+                 is_external_fts=False,
+                 singlenodeMode=False
```

Comment:
unify  `singlenodeMode` like `is_external_fts `, `is_external_fts ` style.

Reply:
The original code is in a mix style.
        addTo.add_option('-U', '--specialMode', type='choice', choices=['maintenance'],
                         metavar='maintenance', action='store', default=None,
                         help=SUPPRESS_HELP)
                         
Since specialMode is a mode and use camel style, I follow it. 😂
After discussion, no change needed.